app/src/main/java/com/eveningoutpost/dexdrip/GcmActivity.java

```diff
@@ -279,6 +279,10 @@ public synchronized static void syncBGReading(BgReading bgReading) {
             UserError.Log.wtf(TAG, "Cannot sync null bgreading - should never occur");
             return;
         }
+        if(Home.get_is_libre_whole_house_collector()) {
```

Comment:
Why is this conditional here? What is it for?

Reply:
In order for an xDrip to use BT to get data it needs to have hardware_data_type == librebt
In order for him to send data it needs to be marked as master.

The question here is what will happen if the user will start a new sensor (and add calibrations).
If he will then the network will have two masters, each sending BG data (as well as other messages).

In order to only have one master on the network, I have limited this master to send messages of LibreBlock, alarm snoozes and 'desert mode' messages. This is why we need the get_is_libre_whole_house_collector condition.

An alternative to this is to write in documentation that the users will not start a new sensor, but then users will find it hard to tell if this xDrip is realy communicating with the miaomiao.